content/zh/docs/tasks/administer-cluster/ip-masq-agent.md

```diff
@@ -0,0 +1,209 @@
+---
+title: IP Masquerade Agent User Guide
+content_template: templates/task
+---
+
+{{% capture overview %}}
+<!--
+This page shows how to configure and enable the ip-masq-agent.
+-->
+此页面展示如何配置和启用 ip-masq-agent。
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture discussion %}}
+<!--
+## IP Masquerade Agent User Guide
+-->
+## IP Masquerade Agent 用户指南
+
+<!--
+The ip-masq-agent configures iptables rules to hide a pod's IP address behind the cluster node's IP address. This is typically done when sending traffic to destinations outside the cluster's pod [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) range.
+-->
+ip-masq-agent 配置 iptables 规则以隐藏位于集群节点 IP 地址后面的 pod 的 IP 地址。 这通常在将流量发送到集群的 pod [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) 范围之外的目的地时使用。
```

Comment:
This links to a page in English-language Wikipedia. Is that intentional?

Reply:
https://zh.wikipedia.org/wiki/%E6%97%A0%E7%B1%BB%E5%88%AB%E5%9F%9F%E9%97%B4%E8%B7%AF%E7%94%B1
here is the corresponding zh page, shall I replace it ?

Reply:
Sounds good to me @jiaj12

Reply:
all done